vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -112,14 +112,17 @@ $global:imagesToPull += @(
     # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
     "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.23",
     # CNS (Container Networking Service) Owner: jaer-tsun (Jaeryn)
-    "mcr.microsoft.com/containernetworking/azure-cns:v1.4.52",
+    "mcr.microsoft.com/containernetworking/azure-cns:v1.4.56",
+    "mcr.microsoft.com/containernetworking/azure-cns:v1.4.54",
     "mcr.microsoft.com/containernetworking/azure-cns:v1.5.35",
-    "mcr.microsoft.com/containernetworking/azure-cns:v1.6.7",
+    "mcr.microsoft.com/containernetworking/azure-cns:v1.5.38",
     "mcr.microsoft.com/containernetworking/azure-cns:v1.6.10",
+    "mcr.microsoft.com/containernetworking/azure-cns:v1.6.11",
     # CNI installer for azure-vnet. Owner: evanbaker
     "mcr.microsoft.com/containernetworking/azure-cni:v1.5.35",
-    "mcr.microsoft.com/containernetworking/azure-cni:v1.6.7",
-    "mcr.microsoft.com/containernetworking/azure-cni:v1.6.10"
+    "mcr.microsoft.com/containernetworking/azure-cni:v1.5.38",https://acs-mirror.azureedge.net/azure-cni/v1.5.35/binaries/azure-vnet-cni-windows-amd64-v1.5.35.zip
```

Comment:
Not familiar with windows setup at all, should this `.zip` link be here?

Reply:
No, removed it.